test/redis/store/factory_test.rb

```diff
@@ -95,18 +96,18 @@
       end
     end
 
-    describe 'when given host Hash and options Hash' do 
+    describe 'when given host Hash and options Hash' do
```

Comment:
same thing here. it's harder to see what you changed in the diff when there's all this extra stuff around.